rules/game_process.tex

```diff
@@ -127,28 +127,27 @@ \subsubsection{Invalid Goal}
 \subsubsection{Indirect Kick}
 \label{sec:indirect_kick}
 
-The attacking team may only score a goal via indirect kick from any restart into \texttt{playing} from a free kick.
+The attacking team may only score a goal via indirect kick from \texttt{Kick-off}, \texttt{Kick-in} and \texttt{Free Kick}.
```

Comment:
Why specifically mention the Kick in? Wouldn't it not be clearer if we state: 
> The attacking team may only score a goal via indirect kick from any re-/start into \texttt{playing}.

Reply:
That was my original plan, but then we need to exclude penalty kick from that. 
Fifa rules have a more specific list of of what kick is direct and indirect. I think for this year we should stay more generic to not add too much load on referees, but I thought this formulation was more in-lines with future evolution of this rule. That said, if the general opinion is to go with the simplified version, I'm fine with that.